packages/react-fetch/src/ReactFetchBrowser.js

```diff
@@ -36,6 +36,14 @@ type Result = PendingResult | ResolvedResult | RejectedResult;
 const nativeFetch = window.fetch;
 const fetchKey = {};
 
+const ReactCurrentDispatcher =
+  React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED
```

Comment:
I think we should probably add an export to experimental instead of this. If we planned on exposing all of these wrappers ourselves, it might make sense to keep it internal but it'll be semi-public quickly so best not to undermine the meaning of it.

readCache is not a great name though. Let's bikeshed?